src/main/java/uk/gov/hmcts/reform/prl/utils/EmailUtils.java

```diff
@@ -56,4 +57,27 @@ public static Map<String, Object> getCommonSendgridDynamicTemplateData(CaseData
 
         return dynamicTemplateData;
     }
+
+    public static String maskEmail(String text, String email) {
```

Comment:
Can we start a new component that has a non-static methods for Email Utilities please?
Then we should add tests against these methods

Reply:
Added new component MaskEmail